geoengine/ml.py

```diff
@@ -30,12 +34,19 @@ class MlModelConfig:
     description: str = "My Ml Model Description"
 
 
-def register_ml_model(onnx_model: SerializableModel,
+def register_ml_model(onnx_model: ModelProto,
```

Comment:
you can remove the SerializableModel class then I guess?

Reply:
[9bea96e](https://github.com/geo-engine/geoengine-python/pull/196/commits/9bea96e2b1118518423c66c7c499bacefe503a83)

Reply:
[152421b](https://github.com/geo-engine/geoengine-python/pull/196/commits/152421b99471daf09897d1c3e29dd86be33f75db)